server/db/seeders/20230202033013-add_api_test_data.js

```diff
@@ -76,11 +79,11 @@ module.exports = {
       role: 'student'
     }], {})
 
-    let comboenrollmentsection1course2 = await queryInterface.bulkInsert('Enrollments', [{
-      userId: comboUser,
-      sectionId: section1course1,
-      role: 'student'
-    }], {})
+    //let comboenrollmentsection1course2 = await queryInterface.bulkInsert('Enrollments', [{
```

Comment:
Maybe don't leave these as comments? Can this be removed or is it being kept to rework?

Reply:
Commented it out in case we ever want a combo user again? I can remove for now

Reply:
Imo its best practice to remove them and then use git history if we want to bring something back but either way is fine here